packages/specs/schema/src/utils/getJsonSchema.ts

```
@@ -11,24 +11,43 @@ import {getJsonEntityStore} from "./getJsonEntityStore.js";
+  const cache: Map<string, any> = CACHES.get(model) || new Map();
+  CACHES.set(model, cache);
+
   const key = getKey(options);
 
   if (!cache.has(key)) {
-    const schema = execMapper("schema", [entity.schema], options);
+    const entity = getJsonEntityStore(model);
+
+    let mapper = "schema";
+    if (entity instanceof JsonParameterStore) {
+      options = {
```

Biome (1.9.4)

[error] Reassigning a function parameter is confusing.

The parameter is declared here:

Use a local variable instead.

(lint/style/noParameterAssign)